teps/0044-decouple-task-composition-from-scheduling.md

```diff
@@ -14,113 +14,138 @@ authors:
-  in a way that can be reused (e.g. in a Pipeline)
-- Add some of [the features we don't have without PipelineResources](https://docs.google.com/document/d/1KpVyWi-etX00J3hIz_9HlbaNNEyuzP6S986Wjhl3ZnA/edit#)
-  to Tekton Pipelines (without requiring use of PipelineResources), specifically the first feature listed in
-  [the doc](https://docs.google.com/document/d/1KpVyWi-etX00J3hIz_9HlbaNNEyuzP6S986Wjhl3ZnA/edit#heading=h.gi1d1dikb39u):
-  **Task adapters/specialization**
+  Tasks together and have control over the pods and volumes required.
+- Provide a mechanism to colocate Tasks that "run things" (e.g. a build)
+with Tasks that fetch inputs (e.g. git clone) or push outputs (e.g. docker push).
 
-### Non-Goals
+## Non-Goals
 
 - Updating the Task CRD to allow Tasks to reference other Tasks
   at [Task authoring time](https://github.com/tektoncd/community/blob/master/design-principles.md#reusability).
   We could decide to include this if we have some use cases that need it; for now avoiding
   this allows us to avoid many layers of nesting (i.e. Task1 uses Task2 uses Task3, etc.)
   or even worse, recursion (Task 1 uses Task 2 uses Task 1...)
-- Completely replacing PipelineResources: we could decide to solve this by improving PipelineResources,
-  or we could add a new feature via this TEP and still continue to support PipelineResources
-  (since they provide [more functionality than just composition](https://docs.google.com/document/d/1KpVyWi-etX00J3hIz_9HlbaNNEyuzP6S986Wjhl3ZnA/edit#))
-- This was previously a use case we were targeting but we've decided to descope the TEP slightly,
-  though if we end up solving this problem as well, that's a bonus:
-  * A user wants to use a Task from the catalog with a git repo and doesn't want the extra
-    overhead of using a Pipeline, they just want to make a TaskRun,
-    e.g. [@mattmoor's feedback on PipelineResources and the Pipeline beta](https://twitter.com/mattomata/status/1251378751515922432))
-    where he wants to checkout some code and [use the kaniko task](https://github.com/tektoncd/catalog/tree/master/task/kaniko/0.1)
-    without having to fiddle with volumes
-
-### Use Cases (optional)
+- Replacing all functionality that was provided by PipelineResources.
+See [TEP-0074](./0074-deprecate-pipelineresources.md) for the deprecation plan for PipelineResources.
+- Creating one-shot TaskRuns directly from Catalog Tasks that require fetching input or pushing output.
+  For example, [@mattmoor's feedback on PipelineResources and the Pipeline beta](https://twitter.com/mattomata/status/1251378751515922432))
+  is that he wants to checkout some code and [use the kaniko task](https://github.com/tektoncd/catalog/tree/master/task/kaniko/0.1)
+  without having to fiddle with volumes. This was previously a use case we were targeting but we've decided to descope the TEP slightly,
+  though if we end up solving this problem as well, that's a bonus.
```

Comment:
at this point im not sure descoping this actually makes sense anymore, since i think it still matches the problem statement - maybe the key difference is that that is specifically in the context of a TaskRun

so im thinking we could get away with removing this entirely, or maybe it would make sense to say we don't have any constraints that say this has to happen within a Task/TaskRun specifically?

Reply:
👍 just going to remove entirely.